backend/app/ai_helpers/routers.py

```diff
@@ -0,0 +1,488 @@
+    methodType: str
+    goal: str
+    outcome: str
+    numVariants: int
+    prior_type: str
+    reward_type: str
+
+
+class CMABContextSuggestionRequest(BaseModel):
+    """Request model for CMAB context suggestion."""
+
+    name: str
+    description: str
+    methodType: str
+    goal: str
+    outcome: str
+    numVariants: int
+    prior_type: str
+    reward_type: str
+
+
+class CMABArmContext(BaseModel):
+    """Model for CMAB arm context."""
+
+    name: str
+    description: str
+    value_type: str
+
+
+class CMABArmsSuggestionRequest(BaseModel):
+    """Request model for CMAB arms suggestion."""
+
+    name: str
+    description: str
+    methodType: str
+    goal: str
+    outcome: str
+    numVariants: int
+    prior_type: str
+    reward_type: str
+    contexts: list[CMABArmContext]
+
+
+api_key = os.environ.get("GEMINI_API_KEY")
+if not api_key:
+    raise HTTPException(status_code=500, detail="Gemini API key not set")
+
+client = genai.Client(api_key=api_key)
+
+
+@router.post("/suggestBaysAB-arms")
+async def suggest_arms(
+    request: baysABArmsSuggestionRequest,
+) -> list[dict[str, Any]]:
+    """Suggest arms for Bayesian A/B test."""
+    # System instruction for arms suggestion
+    system_instruction = (
+        "You are an assistant for a tool that helps social sector "
+        "organizations run digital experiments.\n"
+        "Given the experiment details below, suggest concise names and "
+        "descriptions for each arm (variant) of the experiment.\n"
+        "For Bayesian A/B tests, there are usually two arms: a control "
+        "(existing/baseline) and a treatment (new/changed feature). "
+        "For other experiment types, use the number of variants provided.\n"
+        "For each arm, also suggest reasonable initial values for mu_init "
+        "(mean prior, between 0 and 1 for rates) and sigma_init "
+        "(standard deviation).\n"
+        "Respond ONLY with a valid JSON array. Each array element should be "
+        "an object with keys: name, description, mu_init, sigma_init. "
+        "No explanation, no markdown."
+    )
+
+    user_prompt = (
+        f"Experiment details:\n"
+        f"- Name: {request.name}\n"
+        f"- Description: {request.description}\n"
+        f"- Method type: {request.methodType}\n"
+        f"- Goal: {request.goal}\n"
+        f"- Outcome: {request.outcome}\n"
+        f"- Number of variants: {request.numVariants}\n"
+        f"- Reward type: {request.reward_type}\n"
+    )
+
+    try:
+        response = client.models.generate_content(
+            model="gemini-2.0-flash",
+            contents=[user_prompt],
+            config=types.GenerateContentConfig(
+                system_instruction=system_instruction,
+                max_output_tokens=500,
+                temperature=0.1,
+                response_mime_type="application/json",
+            ),
+        )
+        return json.loads(response.text)
+    except Exception as e:
```

Comment:
Don't catch base exceptions! What exception are you expecting here? Let's just catch those